scripts/parachain-dev-setup.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Spinup a local env with relaychains and parachains for testing.
+# This script expects compiled polkadot, acala and vln binaries under /polkadot, /Acala and /vln-node
+# directories. Run this script only after initial setup as per https://github.com/paritytech/cumulus/blob/master/README.md
+# Ref : https://wiki.acala.network/build/development-guide/composable-chains/open-hrmp-channel
+
+set -e
+# turn a string into a flag
+function flagify() {
+  printf -- '--%s' "$(tr '[:upper:]' '[:lower:]' <<< "$1")"
+}
+
+# start relaychains in seperate screens for debugging
+function startrelaychain() {
+ screen_name="$1"
+ auth="$2"
+ port="$3"
+ wsport="$4"
+ rpcport="$5"
+
+ screen -dmS "$screen_name" \
```

Comment:
How portable is this, is it GNU screen command?

Reply:
yes